lib/diaspora/taggable.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Oh hi there little newline! You look pretty out of place here, but I will help you finding your original destination!